scripts/n8n-backup.sh

```
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+echo "Backing up n8n workflows..."
+
+# Create workflows directory if not exists
+mkdir -p /home/node/.n8n/workflows
+cd /home/node/.n8n/workflows
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)